### ISSUE TITLE
How should Aura2 Design Ideas/Feature Requests be Handled?

### ISSUE THREAD

**Issue by AphonicChaos** [fosskers/aura#249]
**tl;dr** - Should ideas regarding Aura 2's design be submitted as an issue, wiki edit, pull request, or something else?

At the moment, there seems to be at least three ways to request features on github (ie, not including the official Arch Linux thread):
1. Open an issue here
2. Edit the wiki
3. Submit a pull request for the design document

The first doesn't seem to be a problem, except that I've seen it mentioned that users should add the info to the design document. I'm not sure if such suggestions should be taken as "that's a good idea, go ahead and add it to document" or "any new features should be added to the design document". I'd suspect the former, though that then raises the question of whether or not the tex file should be modified or the wiki page.

The issue with the second is that as it's a wiki, there doesn't seem to be any kind of vetting process. That is, because a wiki, anyone can edit it, which is great for accessibility, but bad in that it lends itself to feature creep and disorganization. 

Using PR's would force a sort of structured conversation to happen around the suggested feature. Any discrepancies could be hashed out before finally approving the changes. If you compare that to the current (assumed) process path, we reduce this:

> Open Issue -> Discuss Issue -> get blessing from @fosskers to add to wiki -> Add to Wiki -> Whatever amount of editing needs to happen after the fact

To this:

> Open PR -> Discuss -> Merge or reject

The blessing step and ad-hoc editing is contained within the discuss portion of the process. 

Another alternative might be to do something akin to Python PEPs or RFCs, though I suspect that (at least for now) this would be overkill.
